docs/content/Modpacks/Materials-and-Elements/Material-Properties.md

```diff
@@ -1,58 +1,90 @@
+
 ---
 title: Material Properties
 ---
 
 
-# Material Properties (WIP)
+# Material Properties
 
 ## Blast Furnace Properties
 
```

Comment:
This page starts rather abruptly. Can we get an introduction of what material properties are, how to apply them, etc?